tSQLtCLR/tSQLtCLR/ResultSetToTable.cs

```diff
@@ -0,0 +1,150 @@
+﻿/*
+ * Copyright 2017 Jonathan Hall <jonathan.hall@kjr.com.au>
```

Comment:
@lizbaron, As I understand it that this is correct way to contribute code to an open source project?

The contribution uses the same license as tSQLt [/Build/License.txt](https://github.com/tSQLt-org/tSQLt/blob/main/Build/License.txt#L111-L123)

> [...] You may add Your own copyright statement to Your 
modifications and may provide additional or different license terms and 
conditions for use, reproduction, or distribution of Your modifications, or for 
any such Derivative Works as a whole, provided Your use, reproduction, and 
distribution of the Work otherwise complies with the conditions stated in this 
License.
>
> 5. Submission of Contributions. Unless You explicitly state otherwise, any 
Contribution intentionally submitted for inclusion in the Work by You to the 
Licensor shall be under the terms and conditions of this License, without any 
additional terms or conditions. Notwithstanding the above, nothing herein shall 
supersede or modify the terms of any separate license agreement you may have 
executed with Licensor regarding such Contributions.